packages/node_modules/@webex/plugin-meetings/src/media-checker/index.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/**
+ *
+ * Media Health Checker
+ * @export
+ * @class MediaChecker
+ */
+export default class MediaChecker {
+  /**
+   * Create an instance of MediaChecker.
+   * @param {bool} isAudioSent
+   * @memberof MediaChecker
+   */
+  constructor(isAudioSent) {
```

Comment:
This probably won't have anything passed, but if it does, please pass the `options` argument like in other classes.